framework/caching-cmm.md

```diff
@@ -135,7 +135,8 @@ The number of bytes encrypted by the [encryption](structures.md#encryption-mater
 
 ### Get Encryption Materials
 
-If the [algorithm suite](algorithm-suites.md) requested contains a [Identity KDF](algorithm-suites.md#identity-kdf),
+If the [algorithm suite](algorithm-suites.md) is present in the encryption materials request,
+and it contains an [Identity KDF](algorithm-suites.md#identity-kdf),
```

Comment:
What happens if the algorithm suite isn't present? What does the Caching CMM do?

Reply:
The next paragraph ("Otherwise, the caching CMM...") addresses that: If either the algorithm suite isn't present, or if it is present and doesn't contain an Identity KDF, then the caching CMM checks the cache, and then returns materials if found or delegates to the underlying CMM if not.